File: PythonDataAdapters/stimulsoft_data_adapters/enums/StiDataType.py
"""
Stimulsoft.Reports.JS
Version: 2025.2.4
Build date: 2025.05.19
License: https://www.stimulsoft.com/en/licensing/reports
"""

from enum import Enum


class StiDataType(Enum):

    NONE: None
    JAVASCRIPT = 'text/javascript'
    JSON = 'application/json'
    XML = 'application/xml'
    HTML = 'text/html'
    CSV = "text/csv"
    TEXT = "text/plain"


### Helpers

    @staticmethod
    def getValues():
        return [enum.value for enum in StiDataType if enum.value != None]

File: PHPDataAdapters/enums/StiDataType.php
<?php
# Stimulsoft.Reports.JS
# Version: 2026.1.1
# Build date: 2025.12.17
# License: https://www.stimulsoft.com/en/licensing/reports
?>
<?php

namespace Stimulsoft\Enums;

use Stimulsoft\StiFunctions;

class StiDataType
{
    const JavaScript = "text/javascript";
    const JSON = "application/json";
    const XML = "application/xml";
    const HTML = "text/html";
    const CSV = "text/csv";
    const Text = "text/plain";


### Helpers

    public static function getValues(): array
    {
        return StiFunctions::getConstants("Stimulsoft\Enums\StiDataType");
    }
}